Osiris/Hacks/Misc.h

```diff
@@ -32,7 +32,7 @@ namespace Misc {
     void quickHealthshot(UserCmd*) noexcept;
     void fixTabletSignal() noexcept;
     void fakePrime() noexcept;
-
+    static void usespam(UserCmd*) noexcept;
```

Comment:
**static** is not good here
**static** is good here, for example, because it's only **Netvars.cpp** related: https://github.com/danielkrupinski/Osiris/blob/master/Osiris/Netvars.cpp#L11